deepspeed/module_inject/replace_module.py

```diff
@@ -24,6 +24,8 @@
 
 from .utils import policy_to_ds_container
 
+import gc
```

Comment:
Changes in this file are mainly tracked by #3102 which is to reduce memory consumption in AutoTP